src/loaders/FileLoader.js

```diff
@@ -66,11 +66,12 @@ class FileLoader extends Loader {
 		const req = new Request( url, {
 			headers: new Headers( this.requestHeader ),
 			credentials: this.withCredentials ? 'include' : 'same-origin',
-			// An abort controller could be added within a future PR
 		} );
 
 		// start the fetch
-		fetch( req )
+		fetch( req, {
+			signal: this.abortSignal,
+		} )
```

Comment:
Is there a reason this signal isn't included in the set of options passed into the `Request` above?

Reply:
the signal is not part of the Request object  
see https://developer.mozilla.org/en-US/docs/Web/API/Request and https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters